moma_mission/src/moma_mission/states/transform_visitor.py

```diff
@@ -13,7 +13,7 @@ class TransformVisitorState(StateRosControl):
     def __init__(self, ns):
         StateRosControl.__init__(self, ns=ns)
 
-        self.world_frame = self.get_scoped_param("world_frame", "world")
+        self.control_frame = self.get_scoped_param("control_frame", "odom")
```

Comment:
I don't quite understand yet why in MPC it was called `reference_frame` and here `control_frame`.